main.go

```diff
@@ -47,5 +47,17 @@ func main() {
 		log.Fatal(errors.AddContext(err, "failed to build the api"))
 	}
 
+	// TODO Start a background loop that would scan the DB for either unpinned
+	// 	or underpinned skylinks and pins/unpins them.
```

Comment:
we should make the `underpinned` value an environment variable so that we can increase/decrease the redundancy easily.

Reply:
Hmm... This should be a cluster-wide setting. A DB-based configuration will make more sense, so we can adjust the entire cluster's pinning level simultaneously. Otherwise some servers might think that 1x is enough while others try to get 2x, thus taking way more load onto themselves than they should.

Reply:
agree that it is a cluster wide variable, but it should still be an env var so we don't need to submit a pull request and deploy to change it. 

follow up, just calling it out.